mobile/lib/modules/backup/models/backup_album.model.dart

```diff
@@ -14,7 +14,6 @@ enum BackupSelection {
 class BackupAlbum {
```

Comment:
Can we get rid of this entire file and store all infos in the local album class?

Reply:
We clear local albums on force refresh. Having this data stored in the local albums class results in the backup data getting removed during refresh

Reply:
Good point.